test/system/users_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'application_system_test_case'
+
+class UsersTest < ApplicationSystemTestCase
+  include Devise::Test::IntegrationHelpers
+
+  setup do
+    Rails.application.env_config['omniauth.auth'] = OmniAuth.config.mock_auth[:google_omniauth]
```

Comment:
I dont think this is needed anymore, since we're using the `sign_in` helper method (which is signing in via standard Devise, not Omniauth)

Reply:
Good point, agree. Deleted it.